### PR TITLE
Fix Melee Weapon Attack Speed

### PR DESCRIPTION
**Pull request by dge21**
Description.

Melee weapons got swapped from attacks/second to second/attack but the actual attack rate value didn't get changed so weapons got all jumbled. I'm pretty sure I got them all but I might've missed a few in weird folders

:cl:
- fix: Fixed melee weapons.

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Fun/toys.yml [hunk 1]
```diff
@@ -1387,7 +1387,7 @@
     sprite: Objects/Fun/toys.rsi
     state: foamblade
   - type: MeleeWeapon
-    attackRate: 1.5
+    attackRate: .6
```
Comment (VMSolidus) [inline Resources/Prototypes/Entities/Objects/Fun/toys.yml:1390]:
```suggestion
    attackRate: .6666
```

File: Resources/Prototypes/Entities/Objects/Fun/toys.yml [hunk 2]
```diff
@@ -630,7 +630,7 @@
     sprite: Objects/Fun/ducky.rsi
     state: icon
   - type: MeleeWeapon
-    attackRate: 1.5
+    attackRate: .6
```
Comment (VMSolidus) [inline Resources/Prototypes/Entities/Objects/Fun/toys.yml:633]:
```suggestion
    attackRate: .6666
```

File: Resources/Prototypes/Entities/Objects/Fun/Instruments/instruments_string.yml [hunk 3]
```diff
@@ -67,7 +67,7 @@
         Blunt: 6
         Shock: 1
     bluntStaminaDamageFactor: 1.5
-    heavyRateModifier: 0.75
+    heavyRateModifier: 1.3
```
Comment (VMSolidus) [inline Resources/Prototypes/Entities/Objects/Fun/Instruments/instruments_string.yml:70]:
```suggestion
    heavyRateModifier: 1.3333
```

File: Resources/Prototypes/Entities/Objects/Materials/crystal_shard.yml [hunk 4]
```diff
@@ -20,7 +20,7 @@
   - type: SpaceGarbage
   - type: MeleeWeapon
     wideAnimationRotation: -22.5
-    attackRate: 1.5
+    attackRate: .6
```
Comment (VMSolidus) [inline Resources/Prototypes/Entities/Objects/Materials/crystal_shard.yml:23]:
```suggestion
    attackRate: .6666
```

File: Resources/Prototypes/Entities/Objects/Misc/utensils.yml [hunk 5]
```diff
@@ -53,7 +53,7 @@
     - Fork
   - type: MeleeWeapon
     wideAnimationRotation: 180
-    attackRate: 1.5
+    attackRate: .6
```
Comment (VMSolidus) [inline Resources/Prototypes/Entities/Objects/Misc/utensils.yml:56]:
```suggestion
    attackRate: .6666
```

File: Resources/Prototypes/Entities/Objects/Misc/utensils.yml [hunk 6]
```diff
@@ -100,7 +100,7 @@
     - Spoon
   - type: MeleeWeapon
     wideAnimationRotation: 180
-    attackRate: 1.5
+    attackRate: .6
```
Comment (VMSolidus) [inline Resources/Prototypes/Entities/Objects/Misc/utensils.yml:103]:
```suggestion
    attackRate: .6666
```

File: Resources/Prototypes/Entities/Objects/Tools/cowtools.yml [hunk 7]
```diff
@@ -60,7 +60,7 @@
   - type: Item
     sprite: Objects/Tools/Cowtools/wronch.rsi
   - type: MeleeWeapon
-    attackRate: 1.5
+    attackRate: .6
```
Comment (VMSolidus) [inline Resources/Prototypes/Entities/Objects/Tools/cowtools.yml:63]:
```suggestion
    attackRate: .6666
```

**Review by VMSolidus — CHANGES_REQUESTED** [review]

**Review by angelofallars — CHANGES_REQUESTED** [review]
We also need to adjust the attack speed values of the weapons in the `Nyanotrasen`, `DeltaV` and `WhiteDream` (blood cult) namespaces, and there are many of them. Except for the `LunchboxGeneric` in `DeltaV`, as I already dealt with its attack speed.

**Comment by dge21** [comment]
alright, I think I got everything now

**Review by VMSolidus — APPROVED** [review]

**Comment by OldDanceJacket** [comment]
Did a glance over, everything looks good, thank you a ton.

**Review by OldDanceJacket — APPROVED** [review]

**Pull request merged**